backend/src/nodes/pytorch_nodes.py

```diff
@@ -73,7 +73,7 @@ def __init__(self):
             Real-ESRGAN's SRVGG architecture, and Swift-SRGAN."""
         self.inputs = [PthFileInput()]
         self.outputs = [
-            ModelOutput(kind="pytorch", should_broadcast=True),
+            ModelOutput(kind="PyTorchModel", should_broadcast=True),
```

Comment:
Is this correct?

Reply:
This was the only one I saw using "pytorch" instead of "PyTorchModel" (the default)

Reply:
I think you mixed up type and [kind](https://github.com/joeyballentine/chaiNNer/blob/f3ce6b3dbbe4770bd487a1a29cabc3090190a5cb/src/common/common-types.ts#L55).